.gitpod.yml

```
@@ -0,0 +1,7 @@
+---
+image:
+  file: .gitpod.dockerfile
+
+tasks:
+  - init: |
+      go test ./...
```

```suggestion
      echo "Welcome to TheAlgorithms/Go"
```

I think it might be best not to run tests in the dev environment - the user can decide to run them if they want to.